### DISCUSSION TITLE
MS 4.0.2 — Mixer's Metronome channel M (mute) and S (solo) buttons

### DISCUSSION THREAD

**Discussion by scorster** [musescore/MuseScore#16790] [General]
Can anyone describe the design intent of the Mute and Solo buttons in the Metronome channel strip? Behavior is wonky. 

For instance:

• Toggling Metronome playback also toggles the M button in the Metronome mixer channel, which kinda makes sense. 
• But the reverse does not occur: when the metronome is enabled pressing M (in the mixer's Metronome channel) does not disable/mute the metronome. 

In contrast, Soloing the Metronome track works ... but who does THAT?

Similarly I'm surprised to see M and S in the Master track. And that:

 • S toggles, but has no effect. 
 • M shows a reaction to a click, but neither toggles the button display nor Mutes the Master output.

**Comment by stefhaynes** [comment]
yeah really wish that mute button on the master fader worked...
